index.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html id="main">
+    <head>
+        <meta charset="UTF-8">
+        <title>web witch</title>
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Varela&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="file:///Users/chan/Developer/projects/portfolio/styles.css"> 
```

Comment:
Please use **relative** path instead of absolute path. Absolute path like what you currently have will work for your machine locally as it finds the path to `style.css` starting from your **machine root**. However, when folx need to work on the same project they won't have the same file paths as your machine. Instead **relative** combines paths that are needed **within** the project and will be exactly what other folx need to reference/use the same files.

So for this example, you needed to reference the `styles.css` file like so:
```suggestion
        <link rel="stylesheet" href="styles.css"> 
```

Fixing the path will be very important when it comes to **deployment**. When you deploy your portfolio using heroku or any other service it's like transferring your project onto a virtual machine. That virtual machine will not have the same paths as your local machine either so it also needs a **relative path** in order to find the files/resources for your project.